### PR TITLE
Format help output to fit in 80 columns

### PR DESCRIPTION
**Pull request by g-maxime**
Sample:
```
MediaConch Command Line Interface 17.05
Usage: "MediaConch [-Options...] FileName1 [Filename2...]"

Options:
Help:
--Help, -h
          Display this help and exit
--Help=Advanced, -ha
          Display the advanced help and exit
--Version, -v
          Display the version and exit

--User=UserID, -u UserID
          CLI user will be UserID

Configuration Element:
--Configuration=ConfigurationFile, -c ConfigurationFile
          Use ConfigurationFile as configuration file
          Default, it uses the one in the user data application file
--PluginsConfiguration=PluginsConfigurationFile, -pc PluginsConfigurationFile
          Use PluginsConfigurationFile as plugins configuration file
          Default, it uses the struct in the configuration file or
          no plugins configured if not inside.
Reporting Elements:
--Mediaconch, -mc
          Output MediaConch report (default)
          (MediaConch contains default verbosity of implementationChecks
          plus any provided policy checks.)
--Mediainfo, -mi
          Output MediaInfo report
--Mediatrace, -mt
          Output a trace of the file

Policy Checker:
--Policy=PolicyFileName, -p PolicyFileName
          Apply the policy (XSL or Schematron) 
--CreatePolicy
          Create a policy (XSL) from a file
--PolicyReferenceFile=VideoFile, -prf VideoFile
          Use the VideoFile as reference to compare in policy

Output Formats:
--Format=text -ft
          Output MediaConch reports in Text format
--Format=xml -fx
          Output MediaConch reports in XML format
          (changed to -fa if there are more than 1 tool or more than 1 file)
--Format=html -fh
          Output MediaConch reports in HTML format
--Format=simple -fs
          Output MediaConch reports in a shorten form
          (>1 line only if there are errors), default format
--Format=csv -fc
          Output MediaConch reports in CSV format (only for policies)
--Format=maxml -fa
          Output in MediaArea XML format
--Display=DisplayFileName, -d DisplayFileName
          Apply the display transformation (XSL)

--List
          List files analyzed.

Watch folder:
--WatchFolders-List, -wfl
          List the folder watched
--WatchFolder=folder -wf folder
          Send to the server a folder to watch
--WatchFolder-Reports=folder -wfr folder
          Create reports of the selected watch folder to this folder
--WatchFolder-Not-Recursive
          If watch folder is enabled, do not check recursively the folder
--WatchFolder-User=userId -wfu userId
          Create reports of the watch folder for the selected user ID

Plugins:
--PluginsList
          Output the plugins information ID
--UsePlugin=PluginId, -up PluginId
          By default, only format plugin are used.
          With this command, you can give the plugin ID you want to use.
          Plugin ID can be get using the --pluginslist.

File:
--FileInformation, -fi
       Print files information and quit
```

### CODE REVIEW

**Comment by JeromeMartinez** [comment]
Ping @retokromer

**Comment by retokromer** [comment]
I personally would indent less (e.g. by 4 chars) but also for the `--` level. I can add a commit with a proposal (as soon as I an on a real computer), if you wish.

**Comment by JeromeMartinez** [comment]
I have no strong opinion about it, just not doing something too different compared to others.
But I see every tool does a bit like it wants.

Is it fine with:
```
Configuration Element:
  --Configuration=ConfigurationFile, -c ConfigurationFile
      Use ConfigurationFile as configuration file
      Default, it uses the one in the user data application file
```
(only 4 spaces before description looks weird)

**Comment by retokromer** [comment]
Indeed! (My favourite is indent 2, which I personally use, but today usually 4 is preferred. And the mix you propose is fine with me.)

The idea to have an indent also with `--` is _readability_. One can find quicker the section he/she is looking for, and then the item inside the section. (I’m a frequent user of `-h`s and `man`s.)

**Comment by g-maxime** [comment]
PR updated, new output:

```
MediaConch Command Line Interface 17.05
Usage: "MediaConch [-Options...] FileName1 [Filename2...]"

Options:
Help:
  --Help, -h
      Display this help and exit
  --Help=Advanced, -ha
      Display the advanced help and exit
  --Version, -v
      Display the version and exit

  --User=UserID, -u UserID
      CLI user will be UserID

Configuration Element:
  --Configuration=ConfigurationFile, -c ConfigurationFile
      Use ConfigurationFile as configuration file
      Default, it uses the one in the user data application file
  --PluginsConfiguration=PluginsConfigurationFile, -pc PluginsConfigurationFile
      Use PluginsConfigurationFile as plugins configuration file
      Default, it uses the struct in the configuration file or
      no plugins configured if not inside.
Reporting Elements:
  --Mediaconch, -mc
      Output MediaConch report (default)
      (MediaConch contains default verbosity of implementationChecks
      plus any provided policy checks.)
  --Mediainfo, -mi
      Output MediaInfo report
  --Mediatrace, -mt
      Output a trace of the file

Policy Checker:
  --Policy=PolicyFileName, -p PolicyFileName
      Apply the policy (XSL or Schematron) 
  --CreatePolicy
      Create a policy (XSL) from a file
  --PolicyReferenceFile=VideoFile, -prf VideoFile
      Use the VideoFile as reference to compare in policy

Output Formats:
  --Format=text -ft
      Output MediaConch reports in Text format
  --Format=xml -fx
      Output MediaConch reports in XML format
      (changed to -fa if there are more than 1 tool or more than 1 file)
  --Format=html -fh
      Output MediaConch reports in HTML format
  --Format=simple -fs
      Output MediaConch reports in a shorten form
      (>1 line only if there are errors), default format
  --Format=csv -fc
      Output MediaConch reports in CSV format (only for policies)
  --Format=maxml -fa
      Output in MediaArea XML format
  --Display=DisplayFileName, -d DisplayFileName
      Apply the display transformation (XSL)

  --List
      List files analyzed.

Watch folder:
  --WatchFolders-List, -wfl
      List the folder watched
  --WatchFolder=folder -wf folder
      Send to the server a folder to watch
  --WatchFolder-Reports=folder -wfr folder
      Create reports of the selected watch folder to this folder
  --WatchFolder-Not-Recursive
      If watch folder is enabled, do not check recursively the folder
  --WatchFolder-User=userId -wfu userId
      Create reports of the watch folder for the selected user ID

Plugins:
  --PluginsList
      Output the plugins information ID
  --UsePlugin=PluginId, -up PluginId
      By default, only format plugin are used.
      With this command, you can give the plugin ID you want to use.
      Plugin ID can be get using the --pluginslist.

File:
  --FileInformation, -fi
       Print files information and quit
```

File: Source/CLI/Help.cpp [hunk 1]
```diff
@@ -27,92 +27,95 @@ int Help()
...
+    TEXTOUT("  --Format=simple -fs");
+    TEXTOUT("      Output MediaConch reports in a shorten form");
+    TEXTOUT("      (>1 line only if there are errors), default format");
+    TEXTOUT("  --Format=csv -fc");
+    TEXTOUT("      Output MediaConch reports in CSV format (only for policies)");
+    TEXTOUT("  --Format=maxml -fa");
+    TEXTOUT("      Output in MediaArea XML format");
+    TEXTOUT("  --Display=DisplayFileName, -d DisplayFileName");
+    TEXTOUT("      Apply the display transformation (XSL)");
     TEXTOUT("");
```
Comment (retokromer) [inline Source/CLI/Help.cpp]: and delete this `TEXTOUT("");`

File: Source/CLI/Help.cpp [hunk 2]
```diff
@@ -27,92 +27,95 @@ int Help()
...
-    TEXTOUT("--PluginsConfiguration=PluginsConfigurationFile, -pc PluginsConfigurationFile");
-    TEXTOUT("                    Use PluginsConfigurationFile as plugins configuration file");
-    TEXTOUT("                    Default, it uses the struct in the configuration file or no plugins configured if not inside.");
+    TEXTOUT("  --Configuration=ConfigurationFile, -c ConfigurationFile");
+    TEXTOUT("      Use ConfigurationFile as configuration file");
+    TEXTOUT("      Default, it uses the one in the user data application file");
+    TEXTOUT("  --PluginsConfiguration=PluginsConfigurationFile, -pc PluginsConfigurationFile");
+    TEXTOUT("      Use PluginsConfigurationFile as plugins configuration file");
+    TEXTOUT("      Default, it uses the struct in the configuration file or");
+    TEXTOUT("      no plugins configured if not inside.");
```
Comment (retokromer) [inline Source/CLI/Help.cpp]: sorry, for nit-picking:
add `TEXTOUT("");`

Comment (g-maxime) [inline Source/CLI/Help.cpp]: Added, thanks for the feedbacks.

**Comment by JeromeMartinez** [comment]
Close https://github.com/MediaArea/MediaConch/issues/184

**Pull request merged**